sequencer/sequencer.go

```diff
@@ -203,6 +205,9 @@ func (s *Sequencer) AddProcessID(pid []byte) {
 	if s.pids.Add(pid) {
 		log.Infow("process ID registered for sequencing", "processID", fmt.Sprintf("%x", pid))
 	}
+	if err := s.stg.SetProcessAccpetingVotes(pid, true); err != nil {
```

Comment:
typo: `Accpeting`

Reply:
Lol, I had to read it three times to realize...